cmd/kubeadm/app/util/etcd/etcd.go

```diff
@@ -341,10 +343,25 @@ func (c *Client) RemoveMember(id uint64) ([]Member, error) {
 	return ret, nil
 }
 
-// AddMember notifies an existing etcd cluster that a new member is joining, and
+// AddMember adds a new member into the etcd cluster
+func (c *Client) AddMember(name string, peerAddrs string) ([]Member, error) {
+	return c.addMember(name, peerAddrs, false, false)
+}
+
+// AddMemberAsLeaner adds a new learner member into the etcd cluster
+func (c *Client) AddMemberAsLeaner(name string, peerAddrs string) ([]Member, error) {
```

Comment:
we don't use this function anywhere?

Reply:
I think we can remove it as we don't have a plan to support adding a learner without promoting it.